src/loaders/workspace_uploader/workspace_uploader.py

```diff
@@ -410,24 +410,50 @@ def _query_workspace_with_load_id(
     workspace_id: int,
     load_id: str,
     obj_names: list[str],
-) -> tuple[list[str], list[str]]:
+    assembly_objs_only: bool = True,
+) -> tuple[list[str], list[str], list[str], list[str]]:
```

Comment:
assembly_objs_info and genome_objs_info not necessarily return a list of string?

Reply:
👍